shell/components/ClusterIconMenu.vue

```diff
@@ -106,10 +107,12 @@ export default {
     <i
       v-if="!routeCombo && cluster.pinned"
       class="icon icon-pin cluster-pin-icon"
+      :alt="t('nav.ariaLabel.pinCluster')"
```

Comment:
this one should have some reference to the cluster name

Reply:
fixed